src/components/apply/UrlField.tsx

```diff
@@ -1,17 +1,47 @@
+import { ChangeEvent, useState } from 'react';
+
 import InputField from '../common/input/InputField';
 import Title from '../common/title/Title';
 
-function UrlField() {
+import { APPLY_MESSAGE } from '@/constants/applyMessages';
+import { Question } from '@/types/apis/question';
+import { validateUrlDetail, validateUrlStartHttp } from '@/utils/validateUrl';
+
+interface UrlFieldProps {
+  data: Question;
+  onChange: (id: number, text: string) => void;
+  value: string;
+}
+
+function UrlField({ data, onChange, value }: UrlFieldProps) {
+  const [text, setText] = useState(value ?? '');
```

Comment:
```suggestion
 const [text, setText] = useState<string>(value);
```
대신 사용하신 이유가 text의 초기 상태를 undefined가 아닌 빈 문자열로 처리해주기 위함 인지 궁금합니다~

Reply:
넵 맞습니다! 
props value가 없을 땐 input value로 빈 문자열을 넣어주기 위함입니다